### PR TITLE
Add Adaptive Icon For API 26+

### PR DESCRIPTION
**Pull request by RaafatAkkad**
adaptive icon preview

Tested on API 15, 24, 25, 26.

Adaptive Icons are usable on API 26 or Launchers that support it e.g Nova Launcher

### CODE REVIEW

**Comment by SyncedSynapse** [comment]
Good, this was something that needed to be done soon.

I won't merge this right now because i suspect that making the target v26 might break something else on Kore, namely the background service that handles the notification and the phone listener. The new rules on background services will probably interfere with it so i need to review it before updating the version.

File: art/launcher/sam/final/ic_launcher_foreground.svg [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+       id="path4542" />
+  </g>
+</svg>
```
Comment (SyncedSynapse) [inline art/launcher/sam/final/ic_launcher_foreground.svg]: This file would probably be better located at art/launcher/v26/

File: art/launcher/sam/final/ic_launcher_foreground_shadow.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  <g>
+    <path id="a" d="M97.287912 132.45509c-10.419448-10.49555-20.835398-20.99456-31.251931-31.493 4.74784-4.679293 9.330087-9.531365 14.275995-14.006006 11.85898 11.512635 23.345804 23.398996 35.006924 35.110006-4.74333 4.74-9.48667 9.48-14.23 14.22-1.266997-1.277-2.533993-2.554-3.800988-3.831zm6.892988-34.320006c-5.818988-5.86841-11.722216-11.655412-17.39695-17.665 4.434967-4.82069 9.072125-9.455359 13.89495-13.888 11.94149 11.516482 23.50415 23.415625 35.185 35.194006-4.71278 4.58796-9.35255 9.25189-14.154 13.748-5.90947-5.72896-11.70195-11.57671-17.529-17.389006zM78.784 113.78c-10.419476-10.49554-20.835456-20.99455-31.252-31.493 4.747842-4.67929 9.33009-9.531361 14.276-14.006 11.859014 11.512624 23.345865 23.398974 35.007 35.11-4.743333 4.74-9.486667 9.48-14.23 14.22l-3.801-3.831zm6.893-34.32c-5.818998-5.868417-11.722244-11.655417-17.397-17.665 4.434984-4.820691 9.07216-9.455358 13.895-13.888 11.941491 11.516479 23.50415 23.415622 35.185 35.194-4.71278 4.587956-9.35255 9.251893-14.154 13.748-5.909472-5.728958-11.701953-11.576705-17.529-17.389z"/>
+  </g>
+</svg>
```
Comment (SyncedSynapse) [inline art/launcher/sam/final/ic_launcher_foreground_shadow.svg]: And this one also

File: art/launcher/sam/final/ic_launcher_foreground.svg (hunk 1 above)
Comment (RaafatAkkad) [reply art/launcher/sam/final/ic_launcher_foreground.svg]: Done.

File: art/launcher/sam/final/ic_launcher_foreground_shadow.svg (hunk 2 above)
Comment (RaafatAkkad) [reply art/launcher/sam/final/ic_launcher_foreground_shadow.svg]: Done.

**Comment by RaafatAkkad** [comment]
Agreed, it's a good idea to wait until more devices are on Oreo so that real life scenarios can be tested.

**Comment by SyncedSynapse** [comment]
Hi,

I just took a closer look at this, and the new icon seems a bit bigger than the old one (the inner part obviously). We should keep it the same size.

On Google Now launcher:
device-2017-10-10-183057
device-2017-10-10-183512

**Comment by RaafatAkkad** [comment]
Thanks for the feedback, I've updated the icon.
current vs adaptive
current vs. adaptive

File: .travis.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ android:
     - build-tools-26.0.1
```
Comment (SyncedSynapse) [inline .travis.yml]: This needs to be updated also, otherwise the build fails on Travis.

**Pull request merged**